### ISSUE TITLE
some question

### ISSUE THREAD

**Issue by mahdiyousofun** [Feuerwerk/kryococoa#15]
can i use kryonet in ios by this framework?
and how can i use this framework?

**Comment by Feuerwerk** [comment]
No, i never ported KryoNet. This is only a Objective-C port of the Java Kryo serialization library. And yes it absolutely runs under iOS. In fact, i ported it in the first place to have a counter part of Kryo in my iOS app. But i use a self written library (Java/Objective-C) for RPC which uses Kryo(Cocoa) for the data exchange.
In the reame.md of this repository you find various code samples how to use this library. The first code sample shows how to instantiate Kryo and use it. The other samples show how things work in Objective-C in comparison to Java. The usage of KryoCocoa is quite straight and where it was possible i use exaxtly the same API as Kryo, so when you know how to use it Java you will know how to use it in Objective-C/Swift (with some minor exceptions).
